vflow/convert.py

```diff
@@ -82,6 +83,38 @@ def dict_to_df(d: dict, param_key=None):
     return df
 
 
+def compute_interval(df: DataFrame, d_label, wrt_label, accum=None):
```

Comment:
This def is repeated on line 118. I think we can probably remove this method since it's functionality is covered by `perturbation_stats`, unless we have other ideas for it (e.g., it could return the interval data in a format that will be easy to plot).